Interfaces/MainMenu.xib

```diff
@@ -957,6 +958,19 @@ DQ
                             <menuItem isSeparatorItem="YES" id="1405">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
+                            <menuItem title="Shrink Row" keyEquivalent="" id="E89-Mw-yOH">
```

Comment:
How about `Increase Height` and `Decrease Height` instead of `Shrink/Expand Row`?

If we're going to add commands for changing height then we need to add commands for changing the width as well.